<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="b1">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>110</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="b2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="b3">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>110</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="b6">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>205</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="text">
    <string>6</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="b5">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>205</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="text">
    <string>5</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="b4">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>205</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="text">
    <string>4</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="b9">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>300</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="text">
    <string>9</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="b8">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>300</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="text">
    <string>8</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="b7">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>300</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="text">
    <string>7</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>6</y>
     <width>281</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="erase">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>395</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../images/resources.qrc">
     <normaloff>:/Logo/erase.png</normaloff>:/Logo/erase.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="b0">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>395</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="check">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>395</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../images/resources.qrc">
     <normaloff>:/Logo/check.png</normaloff>:/Logo/check.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../images/resources.qrc"/>
 </resources>
 <connections/>
</ui>
